AppVeyor builds failing in ForensicArtifacts/artifacts

From https://ci.appveyor.com/project/forensicartifacts/artifacts/builds/44064211/job/3s9ffr5e9mty5gdp:

```
If ($isWindows) { .\config\appveyor\install.ps1 }
git : Cloning into '..\l2tdevtools'...
At line:1 char:1
+ git clone https://github.com/log2timeline/l2tdevtools.git ..\l2tdevto ...
+ ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
    + CategoryInfo          : NotSpecified: (Cloning into '..\l2tdevtools'...:String) [], RemoteException
    + FullyQualifiedErrorId : NativeCommandError
 
    Directory: C:\projects\artifacts
Mode                LastWriteTime         Length Name                                                                                                                                                                                
----                -------------         ------ ----                                                                                                                                                                                
d-----         7/4/2022   5:02 AM                dependencies                                                                                                                                                                        
python.exe : [ERROR] Unable to determine package download URLs.
At line:1 char:1
+ & 'C:\Python310-x64\python.exe' ..\l2tdevtools\tools\update.py --down ...
+ ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
    + CategoryInfo          : NotSpecified: ([ERROR] Unable ... download URLs.:String) [], RemoteException
    + FullyQualifiedErrorId : NativeCommandError
 
[ERROR] No packages found.
```

It looks like the update.py script is not able to find packages.

--- a/tools/update.py
+++ b/tools/update.py
@@ -228,7 +228,7 @@ class GithubRepoDownloadHelper(interface.DownloadHelper):
           '(|data-pjax="[^"]*" )(|data-turbo-frame="[^"]*" )'
           'href="([^"]*)"')
       matches = re.findall(expression_string, page_content)
-      
+
       for _, _, match in matches:
         _, _, filename = match.rpartition('/')
         download_url = (

--- a/tools/update.py
+++ b/tools/update.py
@@ -222,10 +222,11 @@ class GithubRepoDownloadHelper(interface.DownloadHelper):
       # * class="js-navigation-open" and class="js-navigation-open " also have
       #   been seen to be used.
       # * an additional data-pjax="{data}" parameter.
-       # * an additional data-turbo-frame="{data}" parameter.
+      # * an additional data-turbo-frame="{data}" parameter.
       expression_string = (
           '<a class="js-navigation-open[^"]*" title="[^"]*" '
-          '(|data-pjax="[^"]*" )(|data-turbo-frame="[^"]*" )href="([^"]*)"')
+          '(|data-pjax="[^"]*" )(|data-turbo-frame="[^"]*" )'
+          'href="([^"]*)"')
       matches = re.findall(expression_string, page_content)
       
       for _, _, match in matches:

--- a/tools/update.py
+++ b/tools/update.py
@@ -222,12 +222,13 @@ class GithubRepoDownloadHelper(interface.DownloadHelper):
       # * class="js-navigation-open" and class="js-navigation-open " also have
       #   been seen to be used.
       # * an additional data-pjax="{data}" parameter.
+       # * an additional data-turbo-frame="{data}" parameter.
       expression_string = (
           '<a class="js-navigation-open[^"]*" title="[^"]*" '
-          '(|data-pjax="[^"]*" )href="([^"]*)"')
+          '(|data-pjax="[^"]*" )(|data-turbo-frame="[^"]*" )href="([^"]*)"')
       matches = re.findall(expression_string, page_content)
-
-      for _, match in matches:
+      
+      for _, _, match in matches:
         _, _, filename = match.rpartition('/')
         download_url = (
             'https://github.com/log2timeline/l2tbinaries/raw/{0:s}/{1:s}/'

